employee_mgmt/app/models/employee.rb

```diff
@@ -1,30 +1,60 @@
 class Employee < ActiveRecord::Base
 
-  def update_phoneno val
+ validates :first_name, presence:true
+  validates :last_name, presence: true
+  validates :phone,allow_blank: true, format: { with: /9[0-9]{9}/,message:"Must have exactly 10 digits number starting with 9"}
```

Comment:
the regex should be a bit different as the given regex will also assume "987987987987" as valid... the regex should be `/9+[0-9]{9}\z/`